.github/workflows/links.yml

```
@@ -0,0 +1,21 @@
+name: links
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  check-links:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - uses: lycheeverse/lychee-action@v1.5.1
+      with:
+        args: >-
+          -v -n "*.md" "**/*.md"
+          --exclude "http://localhost*"
+    - name: fail for link errors
+      run: exit ${{ steps.lychee.outputs.exit_code }}
```

```suggestion
        fail: true
```

see: https://github.com/lycheeverse/lychee-action#passing-arguments